app/models/task.py

```diff
@@ -2,4 +2,23 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String(100))
+    description = db.Column(db.String(200))
+    completed_at = db.Column(db.DateTime, default=None)
+
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'), nullable=True)
+    goal = db.relationship("Goal", back_populates="tasks")
```

Comment:
Flawless connection! 

Fun fact: the default value for nullable is `True` so we don't need to explicitly add it 😊